packages/server/modules/core/migrations/20250630073647_add_embed_tokens.ts

```diff
@@ -0,0 +1,30 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('embed_api_tokens', (table) => {
+    table
+      .string('tokenId')
+      .notNullable()
+      .references('id')
+      .inTable('api_tokens')
+      .onDelete('cascade')
+    table
+      .string('projectId')
+      .notNullable()
+      .references('id')
+      .inTable('streams')
+      .onDelete('cascade')
+    table
+      .string('userId')
+      .notNullable()
+      .references('id')
+      .inTable('users')
+      .onDelete('cascade')
+    table.string('modelIds').notNullable()

```

Comment:
why not use a string array column for this? better DX

Reply:
+1 for next time when it is a string array, but encoded `resourceIdString` as string here